learning/tour-of-beam/frontend/lib/components/sign_in/sign_in_button.dart

```diff
@@ -18,9 +18,8 @@
 
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
-import 'package:playground_components/dismissible_overlay.dart';
+import 'package:playground_components/playground_components.dart';
 
-import '../../constants/sizes.dart';
 import 'sign_in_overlay_content.dart';
 
 class SignInButton extends StatefulWidget {
```

Comment:
All three are of different height:
![image](https://user-images.githubusercontent.com/44893228/188480258-1dc8ac24-bae8-43c4-b875-e713934c4029.png)
![image](https://user-images.githubusercontent.com/44893228/188480310-8ead98c6-6c5f-4b5d-8d59-8566a5912a76.png)

Reply:
Done.